tasks/Task.cpp

```diff
@@ -143,93 +153,44 @@ void* Task::startTaskFunc(void* argument)
         return 0;
     }
 
-    // Prepare the LibManager and required configuration files
-    libManager = new lib_manager::LibManager();
-
-    std::string corelibsConfigPath;
-    // If the graphical interface should be disabled, the configuration 
-    // needs to exclude the shared library with graphics options
-    // Thus, the nogui configuration file needs to be used for the startup 
+    // todo: change this in a configuration of MARS
     if(marsArguments->enable_gui)
     {
-        corelibsConfigPath = marsArguments->config_dir + "/core_libs.txt";
+        std::string cmd;
+        cmd = "cp ";
+        cmd += marsArguments->config_dir + "/core_libs.txt.example ";
```

Comment:
I would prefer either a separation within mars for this. So that mars loads different "default" libraries with a option (internally to mars) instead of moving files around. But this will be a extension to mars.